Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1461,51 +1467,143 @@ extension BrowserViewController: WKUIDelegate {
         return tab.webView
     }
 
+    func createDisableDialoguePrompt(alertController: UIAlertController) {
```

Comment:
I think it would be better if we could move this into BVC; `Browser` is more of a browser model than the view.